### PR TITLE
Enlarge timeout to prevent health check failure

### PR DESCRIPTION
**Pull request by smellthemoon**
Signed-off-by: lixinguo [xinguo.li@zilliz.com](mailto:xinguo.li@zilliz.com)
/kind bug
#25102

### CODE REVIEW

File: internal/proxy/look_aside_balancer.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func (b *LookAsideBalancer) checkQueryNodeHealthLoop(ctx context.Context) {
 			now := time.Now().UnixMilli()
 			b.metricsUpdateTs.Range(func(node int64, lastUpdateTs int64) bool {
 				if now-lastUpdateTs > checkQueryNodeHealthInterval.Milliseconds() {
-					ctx, cancel := context.WithTimeout(context.Background(), 100*time.Millisecond)
+					ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
```
Comment (jiaoew1991) [inline internal/proxy/look_aside_balancer.go:162]: make it configurable

File: pkg/util/paramtable/component_param.go [hunk 2]
```diff
@@ -908,6 +909,16 @@ func (p *proxyConfig) init(base *BaseTable) {
 	}
 	p.TimeTickInterval.Init(base.mgr)
 
+	p.HealthCheckTimetout = ParamItem{
+		Key:          "proxy.healthCheckTimetout",
+		Version:      "2.2.0",
```
Comment (weiliu1031) [inline pkg/util/paramtable/component_param.go:914]: wrong Version field

**Comment by smellthemoon** [comment]
rerun ut

**Pull request merged**